src/FetchyeReduxProvider.jsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { useSelector, useDispatch } from 'react-redux';
+import { defaultFetcher } from './defaultFetcher';
+import { FetchyeContext } from './FetchyeContext';
+
+const defaultEqualityChecker = (a, b) => a.data === b.data
```

Comment:
This is the same as the one in `FetchyeProvider`. Can we move this out for reuse?

Reply:
fixed in https://github.com/americanexpress/fetchye/pull/10/commits/c0839e20c260f67514744ed718caeeb9a7f77396